spec/features/app_activities_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.feature 'App activities' do
+  include AppSpecHelper
+
+  scenario 'list app activities' do
+    with_app do |app|
+      u = UpdateAppService.new(create(:user),
+                           app, { name: 'new name' }).execute
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.